packages/terra-consumer-layout/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "terra-consumer-layout",
+  "main": "lib/Layout.js",
+  "version": "0.0.0",
+  "description": "A re-usable layout component that puts together our navigation",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-consumer.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Consumer",
+    "HealtheLife",
+    "terra-consumer-layout",
+    "Layout",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-consumer/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-consumer#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-router-dom": "^4.2.2",
+    "terra-base": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react-intl": "^2.3.0",
+    "terra-base": "^1.0.0",
```

Comment:
can you bump all of the terra dependencies to the latest version? They remove reset styles which mess portal up